addons/html_editor/static/src/main/font/font_plugin.js

```diff
@@ -520,4 +523,33 @@ export class FontPlugin extends Plugin {
     updateFontSizeSelectorParams() {
         this.fontSize.displayName = this.fontSizeName;
     }
+
+    handleInsertWithinPre(insertContainer, block) {
+        if (block.nodeName !== "PRE") {
+            return insertContainer;
+        }
+        for (const cb of this.getResource("before_handle_insert_within_pre_processors")) {
```

Comment:
I know it's to match the name of the function, but I would nevertheless rename this to `before_insert_within_pre_processors`.